packages/terra-toolkit-docs/README.md

```diff
@@ -0,0 +1,26 @@
+# Eslint Config Terra
+
+[![NPM version](https://badgen.net/npm/v/@cerner/terra-toolkit-docs)](https://www.npmjs.org/package/@cerner/terra-toolkit-docs)
+[![Build Status](https://badgen.net/travis/cerner/terra-toolkit)](https://travis-ci.com/cerner/terra-toolkit)
+
+Contains documentation for packages in the terra-toolkit monorepo
+
+## Versioning
+
+terra-toolkit-docs is considered to be stable and will follow [SemVer](http://semver.org/) for versioning.
+
+1. MAJOR versions represent breaking changes
+2. MINOR versions represent added functionality in a backwards-compatible manner
+3. PATCH versions represent backwards-compatible bug fixes
+
+Consult the component CHANGELOGs, related issues, and PRs for more information.
```

Comment:
this can be removed right?

Reply:
it's still useful, for example: https://www.npmjs.com/package/@cerner/eslint-config-terra#versioning